dev-python/pipenv/pipenv-2021.5.29-r1.ebuild

```
@@ -4,7 +4,7 @@
 EAPI=7
 
 DISTUTILS_USE_SETUPTOOLS=rdepend
-PYTHON_COMPAT=( python3_{7,8,9} )
+PYTHON_COMPAT=( python3_{7,8,9,10} )
```

```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```

The `..` syntax is not obligatory but IMO it's nicer.

---

I agree :-)